### DISCUSSION TITLE
Problem with opening .osc file

### DISCUSSION THREAD

**Discussion by matheusGariglio** [mtex-toolbox/mtex#1471] [Ask Anything] [answered]
Hi, 

I'm trying to open a .osc file in mtex, but apparently all the data in marked as "notIndexed" and I get the following error message : 

'Warning: more column data was passed in than expected. Check your column names make sense!'

It doesn't recognize the phases or any information from the file.

I'm usually using ctf of ang files and I never had a similar problem with those.

Did anyone had this problem opening OSC files and do you know how could I solve it, please?

You can find the file i'm trying to open in the following link:
https://cloud.mines-paristech.fr/index.php/s/LL0MrGXkznOXTfh/download

Cheers,

Matheus

**Comment by lfgmorales** [comment, marked as answer]
Hi Matheus

It is possible that this specific version of *.osc is not compatible with mtex. Can you export this file as *.ang? This always works

cheers


Luiz

**Comment by matheusGariglio** [comment]
Hi Luiz,

Ok, muito obrigado ;)

I'm gonna check with my co-worker if he can export the file in *.ang format.

Cheers,

Matheus
